django-stubs/test/client.pyi

```diff
@@ -6,6 +6,7 @@ from json import JSONEncoder
 from re import Pattern
 from types import TracebackType
 from typing import Any, Generic, Literal, NoReturn, TypedDict, TypeVar, type_check_only
+from urllib.parse import _QueryType
```

Comment:
Is it okay to import this from typeshed?

https://github.com/python/typeshed/blob/8bc9a6536775465280a42745940575d2f5980078/stdlib/urllib/parse.pyi#L145C1-L147

Because ultimately `query_params` works by passing down to `urlencode()`. But maybe that's too broad and unspecific, and we should just go with `Mapping[Any, Any]` ?

Reply:
Not really, it can be changed / renamed / removed in any typeshed version. it is better to copy-paste, in my opinon.

Reply:
I went with `Mapping[Any, Any]`.